package.json

```diff
@@ -52,7 +52,9 @@
     "start": "ng serve",
     "test": "nx run-many --target=test --all --parallel",
     "update": "nx migrate latest",
-    "workspace-generator": "nx workspace-generator"
+    "workspace-generator": "nx workspace-generator",
+    "take": "node ./dx/create-branch-from-issue.js",
+    "pr": "node ./dx/create-pr.js"
```

Comment:
suggestion: consider using [`ts-node`](https://github.com/TypeStrong/ts-node) so we can use TypeScript.